metrotranscom/doorway
# fix: update translations for new features

# Pull Request Template

## Issue Overview

This PR addresses #520 


## Description

Updates translations for new features. There are some translations missing from task file, but i couldn't find them in app.
I'll double check with Sarah, so there might be some additional small changes.

## How Can This Be Tested/Reviewed?

Click through `get-started`,`faq` in different languages, make sure there is no translations missing (or check if formatting is off).
You can also check application flow in different languages (as i couldn't really make this work locally for non eng language).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
